docs/framework/tutorial/install.md

```diff
@@ -5,6 +5,40 @@ short-description: Installing the tools needed for the walkthrough
 
 This section will walk through installing the tools needed for this walkthrough on Endless OS.
 
+## Installing Flatpak ##
+
+You will need Flatpak on your system to complete this walkthrough.
+If you're using Endless OS, it is already included.
+For other Linux distributions, check [Flatpak's installation instructions][flatpak-install].
+
+> **NOTE:** It's possible to complete the walkthrough and make an app without Flatpak, but you're on your own — that's outside the scope.
```

Comment:
I'd replace `that's outside the scope` with `packaging and installing an application using a different file format is outside the scope of this tutorial`.